cabot/metricsapp/models/base.py

```diff
@@ -74,6 +74,14 @@ def update_url(self):
                   'update the check?'
     )
 
+    # TODO(evan): how to enforce consecutive_failures > 0 ?
```

Comment:
you can use `PositiveIntegerField` (example: https://github.com/Affirm/cabot/blob/master/cabot/cabotapp/models.py#L692)

Reply:
Excellent!  Thanks.